nntrainer/src/tensor.cpp

```diff
@@ -634,22 +647,22 @@ int Tensor::argmax() {
 
 float Tensor::l2norm() const {
   float sum = 0.0;
-  for (int i = 0; i < len; i++) {
+  for (unsigned int i = 0; i < dim.getDataLen(); i++) {
```

Comment:
I think it is better to 
```cpp
len = dim.getDataLen();
for (unsigned int i = 0; i < len; i++) { 
 ...
}
```
since getDataLen does multiplication 4 times for later PR.

Reply:
yes. It's better. fixed. Thanks.